test/cea/mp4_cea_parser_unit.js

```diff
@@ -23,21 +23,28 @@ describe('Mp4CeaParser', () => {
   });
 
   it('parses cea data from mp4 stream', () => {
-    let cea708Packets;
     const cea708Parser = new shaka.cea.Mp4CeaParser();
 
-    const expectedCea708Packets = new Uint8Array([181, 0, 49, 71, 65, 57,
-      52, 3, 206, 255, 253, 148, 32, 253, 148, 174, 253, 145, 98, 253, 115,
-      247, 253, 229, 186, 253, 145, 185, 253, 176, 176, 253, 186, 176, 253,
-      176, 186, 253, 176, 49, 253, 186, 176, 253, 176, 128, 253, 148, 44,
-      253, 148, 47, 255]);
+    const expectedCea708Packet = new Uint8Array([
+      0xb5, 0x00, 0x31, 0x47, 0x41,
```

Comment:
If this were a hex dump in a terminal, you would normally see groups of 8 or 16, rather than 5 bytes per line.  I bet 8 would fit here.  I know it seems arbitrary, but would you please reformat into 8 bytes per line?